swap_meet/clothing.py

```diff
@@ -1,2 +1,10 @@
-class Clothing:
-    pass
+from swap_meet.item import Item 
+
+class Clothing(Item): 
+    def __init__(self, condition = 0): 
+        # super().__init__(category = "Clothing")
+        ## super().__init__(condition)
+        self.category = "Clothing"
+        self.condition = condition 
```

Comment:
great! lines 7 and 8 totally work, though we could definitely use the parent class' constructor instead! It already creates `category` and `condition` attributes for us, so let's look at that:

```suggestion
        # super().__init__(category = "Clothing", condition)
```

and I like that you hardcoded "Clothing" since of course this child class Clothing will _always_ be category "Clothing." No reason to give the user the opportunity to mess that up haha